src/codec_aom.c

```diff
@@ -816,6 +816,13 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         unsigned int cqLevel = (cfg->rc_min_quantizer + cfg->rc_max_quantizer) / 2;
         aom_codec_control(&codec->internal->encoder, AOME_SET_CQ_LEVEL, cqLevel);
     }
+#if defined(AOM_CTRL_AV1E_SET_SKIP_POSTPROC_FILTERING)
+    if (cfg->g_usage == AOM_USAGE_ALL_INTRA) {
+        // Enable AV1E_SET_SKIP_POSTPROC_FILTERING for still-picture encoding, which is
+        // different from libaom's default.
+        aom_codec_control(&codec->internal->encoder, AV1E_SET_SKIP_POSTPROC_FILTERING, 1);
+    }
+#endif
```

Comment:
Please move this code earlier, either before line 735, or between line 752 and line 753. You will need to add `#if defined(AOM_USAGE_ALL_INTRA)` around this code.

Reply:
Done.